src/System.CommandLine/Builder/CommandLineBuilder.cs

```diff
@@ -14,6 +14,33 @@ namespace System.CommandLine
     /// </summary>
     public class CommandLineBuilder 
     {
+        /// <inheritdoc cref="CommandLineConfiguration.EnablePosixBundling"/>
+        internal bool EnablePosixBundling = true;
```

Comment:
Because it's a field, the naming convention would typically be `_enablePosixBundling`.

Reply:
@jonsequitur for now I would prefer to stay with the current name, as I expect that next week we are going to introduce mutable `CommandLineConfiguration` and just turn these fields into public properties